src/components/Card/types.tsx

```diff
@@ -0,0 +1,5 @@
+export type K = {
```

Comment:
that's rudely unprofessional and inconvenient...what difference this type makes from the type `any` then? that's the purpose?